src/xclim/core/indicator.py

```diff
@@ -2007,3 +2007,10 @@ def _merge_attrs(dbase, dextra, attr, sep):
             load_locale(loc_dict, locale)
 
     return mod
+
+
+class StandardizedIndexes(ResamplingIndicator):
+    """Resampling but flexible inputs indicators."""
+
+    src_freq = ["D", "MS"]
+    context = "hydro"
```

Comment:
@aulemahal any opposition to this reorginazation? Previously I defined it in _precip, but now it's also needed in _hydrology